### PR TITLE
adds proposal for charter submission & review per #200

### PR DESCRIPTION
**Pull request by jorydotcom**
Adds a charter submission and review process to address this need for project onboarding.

### CODE REVIEW

File: proposals/stage-1/CHARTER_REVIEW/charter_review.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+## Reviewing a charter
+
+Project Charters are the responsibility of the CPC. When reviewing a charter, the CPC shall evaluate:
+
+1. The completeness of the document - all sections marked required in the PROJECT_CHARTER_TEMPLATE.md are filled out. Optional sections that have not been filled out are marked as intentionally omitted.
+1. The comprehensibility of the document - a person familiar with the field but unfamiliar with the project should be able to read the charter and have a sound understanding of where it fits in the ecosystem. 
+1. The accuracy of the document - the document should reflect the reality of the project. It should avoid hyperbolic language (e.g. "world's best developer experience"). 
+1. The lifespan of the document - the CPC should take note of what factors may cause the charter document to change. If data that may change frequently appears in the charter - such as a list of names - the CPC may ask to remove it. 
+
+If the project handles PII, deals with a new intersection between technology and law, or would simply feel more comfortable, they or the CPC Board Representative may request review from Foundation legal counsel. This is an optional step.
```
Comment (mcollina) [inline proposals/stage-1/CHARTER_REVIEW/charter_review.md]: what is PII?

File: proposals/stage-1/CHARTER_REVIEW/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+## Who would be responsible?
+
+The CPC is responsible for reviewing and approving project charters. 
+
+Should legal review of a charter be requested, the CPC Board Representative is responsible for handling that request.
+
+## How would success be measured?
+
+* speed and thoroughness of charter review - we have ~30 project charters to assess in the coming months related to onboarding
+* ease of review - the process should be relatively frictionless where there are no concerns
```
Comment (mcollina) [inline proposals/stage-1/CHARTER_REVIEW/README.md]: I think we should track progress with some tool, 30 are a LOT!

File: proposals/stage-1/CHARTER_REVIEW/charter_review.md (hunk 1 above)
Comment (ljharb) [reply proposals/stage-1/CHARTER_REVIEW/charter_review.md]: Personally Identifiable Information

File: proposals/stage-1/CHARTER_REVIEW/charter_review.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+## Approving a charter
+
+Project Charters should be reviewed and approved within two weeks of submission. An exception may be made if the charter is submitted during a holiday period (such as the end of the year) or in the event either the project or CPC has requested legal review. 
+
+Charters are approved when:
+
+* Reviewing CPC members have added themselves as reviewers on the issue and approved.
+* More than 50% of voting CPC members have approved, OR 
+* More than 2/3 of all CPC members have approved, OR
+* A period of 14 days has elapsed with no concerns or objections raised.
```
Comment (mhdawson) [inline proposals/stage-1/CHARTER_REVIEW/charter_review.md]: This https://github.com/openjs-foundation/cross-project-council/blob/master/GOVERNANCE.md#aproving-project-charters in the governance doc says:

The request is approved when:

there are no outstanding objections
there are two or more approvals by voting CPC members
the board has been consulted in the case of substantial changes
The issue has been open for at least 14 days

I'd suggest we use that, and possibly just link to that so we don't need to maintain in 2 places

**Comment by MylesBorins** [comment]
Seems like this proposal got forgotten about. Considering we are already approving of charters do we still need this formalized to this extent?

File: proposals/stage-1/CHARTER_REVIEW/charter_review.md (hunk 1 above)
Comment (jorydotcom) [reply proposals/stage-1/CHARTER_REVIEW/charter_review.md]: I updated this to explain the acronym!

**Comment by mhdawson** [comment]
@jorydotcom link looks perfect.

**Pull request merged**